rule/utils.go

```diff
@@ -140,3 +140,81 @@ func isCallToExitFunction(pkgName, functionName string) bool {
 func newInternalFailureError(e error) []lint.Failure {
 	return []lint.Failure{lint.NewInternalFailure(e.Error())}
 }
+
+// isUpper checks if rune is digit
+func isDigit(r rune) bool {
+	return r >= '0' && r <= '9'
+}
+
+// isUpper checks if rune is ascii upper case letter
```

Comment:
```suggestion
// isUpper checks if rune is ASCII upper case letter
//
// # Note
// We restrict to A-Z because [Unicode.IsUpper] return true for a large variety of letter
```